config/app.config

```diff
@@ -9,4 +9,7 @@ general_project_directory: tests/resources/projects
 staging_directory: /tmp/
 project_links_directory: /tmp/
 path_to_mover: '/usr/local/mover/1.0.0/'
+dds_conf:
+  token_path: auth.token
```

Comment:
Hmm, how do you mean here? The path on miarka will be defined in `miarka-provision` right?

Reply:
As I understand it, this `token_path` points to a single token file, but since dds requires personal tokens it would be more convenient if the config variable was a path to the folder where the tokens resides and to supply the user/token name as a query parameter part of the request.

So my idea was that it would look something like this:
```
<path to dds tokens directory>
                  |__ user1 
                  |__user2
``` 

We could ofc also just have a specific token file that the user overwrites with their personal token. It will however make it hard for multiple users to do deliveries simultaneously.

Reply:
Aha ok now I understand, will fix that, thanks :)  :+1:

Reply:
Actually I've been thinking, if we are going to let the user specify where their authentication token is upon delivery, do we need a dds tokens directory at all?

Reply:
I was thinking that it would be easier to just pass the username instead of the whole path, that is ofc also doable. In the future, when we implement personal users in Stackstorm the parameter would be filled in automatically by the delivery workflows in snpseq_packs. So I thought that would be neat. 
But it is a good point, as it is now the user still have to generate the token manually, so they will have path anyway. Also specifying the whole path might make it more flexible in a way? 

I'm ok with going with that :+1: